thetis/options.py

```diff
@@ -245,6 +245,12 @@ def __init__(self):
 
         Bottom stress is :math:`\tau_b/\rho_0 = -g \mu^2 |\mathbf{u}|\mathbf{u}/H^{1/3}`
         """
+        self.wd_alpha = None
+        r"""Coefficient or None: Wetting-drying parameter :math:`\alpha`
+
+        Used in bathymetry displacement function that ensures positive water depths. Unit is meters.
+        Default is None, which disables wetting and drying.
```

Comment:
Is setting a default value for `wd_alpha` just the cleanest way to do this, or can we force the user to specify a value? Any default value we set is likely to be inappropriate (and potentially unstable if too small), because the appropriate choice of `wd_alpha` depends on the mesh and bathymetry.

Could we have something like `self.wd_alpha = None` as the default, and then `if wetting_and_drying: assert(wd_alpha is not None)` somewhere?

Or do we just assume that the user has read the docs and knows that they shouldn't leave the default value unchanged?